cmd/state/internal/cmdtree/export.go

```diff
@@ -109,3 +109,40 @@ func newAPIKeyCommand(prime *primer.Values) *captain.Command {
 			return apikey.Run(params)
 		})
 }
+
+func newExportConfigCommand(prime *primer.Values) *captain.Command {
+	config := export.NewConfig(prime)
+	params := export.ConfigParams{}
+
+	return captain.NewCommand(
+		"config",
+		locale.T("export_config_cmd_description"),
+		[]*captain.Flag{
+			{
+				Name: "filter",
+				Description: locale.Tr(
+					"export_config_flag_filter_description",
+					export.RecognizedFilters(),
+				),
+				Value: &params.Filter,
+			},
+		},
+		[]*captain.Argument{},
+		func(ccmd *captain.Command, _ []string) error {
+			return config.Run(ccmd, params)
+		})
+}
+
+func newDirCommand(prime *primer.Values) *captain.Command {
```

Comment:
D'oh, we had two ACs covering the same use-case. Please drop the `dir` subcommand and use only the `--filter`. I'll update the ACs.